frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -45,28 +45,42 @@ const columns = [
   },
 ];
 
-const transformRowData = (mine) =>
-  mine.mine_permit[0] &&
-  mine.mine_permit.map((permit) => ({
-    key: permit.permit_guid,
-    lastAmended: String.EMPTY_FIELD,
-    permitNo: permit.permit_no ? permit.permit_no : String.EMPTY_FIELD,
-    permittee: permit.permittee[0] ? permit.permittee[0].party.party_name : String.EMPTY_FIELD,
-    firstIssued: permit.issue_date ? permit.issue_date : String.EMPTY_FIELD,
+const groupPermits = (permits) =>
+  permits.reduce((acc, permit) => {
+    acc[permit.permit_no] = acc[permit.permit_no] || [];
+    acc[permit.permit_no].push(permit);
+    return acc;
+  }, {});
```

Comment:
Accumulator. Accumulator and value are the standard terms for a fold/reduce. I always start with those and then make a more specific name if needed. Accumulator felt appropriate to leave in here since we're not doing anything overly business-logic-y.